embassy-macros/src/lib.rs

```diff
@@ -342,6 +342,15 @@ pub fn main(args: TokenStream, item: TokenStream) -> TokenStream {
 
     let chip_setup = chip::generate(&embassy_prefix, config);
 
+    #[cfg(feature = "defmt")]
+    let spawn = quote! {
+        unwrap!(spawner.spawn(__embassy_main(spawner, p)));
```

Comment:
This should somehow ensure the `unwrap!` macro is in-scope in the user's code, I'm not sure if it's doing it currently.

Reply:
I'll admit that I'm not sure how to do this.